### PR TITLE
Fixes Issue #6 Adds infra_only and app_log_content_access variables to the installer…

### PR DESCRIPTION
**Pull request by dannywillford**
Adds infra_only and app_log_content_access variables to the installer script for applications that require non-default values. #6

### CODE REVIEW

**Comment by holms** [comment]
Can this be accepted please??

File: tasks/main.yml [hunk 1]
```diff
@@ -39,6 +39,6 @@
    when: agent_installed.stat.exists == False and (oneagent_installer_script_url is defined) and (oneagent_installer_script_url is not none)
 
  - name: "Install Dynatrace OneAgent"
-   shell: "sh /tmp/dynatrace-oneagent.sh"
+   shell: "sh /tmp/dynatrace-oneagent.sh APP_LOG_CONTENT_ACCESS={{ dynatrace_app_log_content_access }} INFRA_ONLY={{ dynatrace_infra_only }}"
```
Comment (holms) [inline tasks/main.yml]: I'd actually would be use `environment` directive: https://docs.ansible.com/ansible/2.6/user_guide/playbooks_environment.html

Comment (MarcinZejer) [reply tasks/main.yml]: Hi @holms I verified with agent installer team that those environment variables will not be read so currently it is the only option to pass them.

**Comment by MarcinZejer** [comment]
Hi @dannywillford I verified with agent installer team that setting INFRA_ONLY=0 as default may not work for some older tenants so I cannot merge the changes. The option would be that only if INFRA_ONLY is set, the parameter is added to command line. This problem is not applicable to APP_LOG_CONTENT_ACCESS so it can be added with default value.

File: tasks/main.yml (hunk 1 above)
Comment (holms) [reply tasks/main.yml]: I've also had problems with "environment" directive :) ended up doing the same

**Comment by holms** [comment]
Putting it as a default probably bad idea. I've actually gave up on dynatrace logging. That dashboard is totally nonusable, it tries to download files, there's limit for file download. I mean it should be elastic search endpoint with kibana like all normal logging solutions does, like papertrail and logentries.

**Pull request merged**